hosts.yaml

```diff
@@ -161,27 +173,39 @@ all:
           hosts:
             cisco-19-42:
               ansible_host: 10.0.19.42
+              calculate_rack: true
             cisco-17-42:
               ansible_host: 10.0.17.42
+              calculate_rack: true
           vars:
             ansible_password: "{{ moc_cisco_password }}"
             ansible_network_os: cisco.nxos.nxos
       vars:
         ansible_user: admin
         ansible_connection: ansible.netcommon.network_cli
+        calculate_rack: false
+        location: Row 3 Pob B
     ipmi_switches:
       children:
         brocade_icx:
           hosts:
             ipmi-cage3:
               ansible_host: 10.0.3.44
+              calculate_rack: true
```

Comment:
It looks like all members of the `brocade_icx` group have `calculate_rack: true`, so we could just this make a variable on the group instead of per-host.

Reply:
cool, done!